cloud_registry/version.py

```diff
@@ -1,3 +0,0 @@
-"""Single source of truth for package version."""
```

Comment:
deleted this file because pyproject config does not support reading version from python variable